modules/040-node-manager/docs/USAGE.md

```diff
@@ -69,3 +69,41 @@ spec:
         value: system
   nodeType: Static
 ```
+
+## An example of install crane on every node
```

Comment:
It does not seem to be a real-world example to me. Could you provide something more realistic? It would be perfect if it were a script for a node group type, e.g., master, frontend.

Maybe kubectl plugins installation?

Reply:
changed to install of cert-manager plugin